stl/inc/yvals_core.h

```diff
@@ -1546,5 +1546,9 @@ compiler option, or define _ALLOW_RTCc_IN_STL to acknowledge that you have recei
 #define _STL_INTERNAL_STATIC_ASSERT(...)
 #endif // _ENABLE_STL_INTERNAL_CHECK
 
+#ifndef _MSVC_CONSTEXPR // TRANSITION, VS2022v17.3p2
+#define _MSVC_CONSTEXPR

```

Comment:
Can we move that to the other custom attributes? Even if it has no value right now?

Reply:
Does it _need_ to be now, or would a followup be ok? I plan a followup to better integrate this with #2649 (move `_HAS_MSVC_ATTRIBUTE` to VCRuntime so it's used consistently for all of the `_MSVC_MEOW` attribute macros) and I'll bring it all together then.

Reply:
Not a hill I would die on.

It is a bit icky to add something with the goal to clean it up afterwards, but I can appreciate that it has a dependency on VCRuntime so its fine by me

Reply:
> It is a bit icky to add something with the goal to clean it up afterwards, but I can appreciate that it has a dependency on VCRuntime so its fine by me

I'm also trying to avoid repeated CI cycles as things move back-and-forth between PRs, and trying to keep the compiler Devs unblocked. This is correct but not in the final form we'd like, which I'm happy to call good enough to decouple future library work from the ongoing compiler PR.